detox/src/devices/Device.js

```diff
@@ -135,6 +135,36 @@ class Device {
     await this.deviceDriver.waitForBackground();
   }
 
+  async enrollBiometrics() {
```

Comment:
Let's please merge these as `setBiometricEnrollment(bool)` across the API stack.

Thanks